draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2899,81 +2712,35 @@ continue.</t>
 <section anchor="sec-lite-conclude" title="Procedures for Lite Implementations">
 
 <t>
-Concluding ICE for a lite implementation is relatively
-straightforward. There are two cases to consider:
-<list style="empty">
-<t>The implementation is lite, and its peer is full. </t>
-<t>The implementation is lite, and its peer is lite. </t>
-</list></t>
-
-<t>
-The effect of ICE concluding is that the agent can free any allocated
-host candidates that were not utilized by ICE, as described in <xref
-target="sec-freeing"/>.
+When ICE concludes, a lite agent can free host candidates that were
+not used by ICE, as described in <xref target="sec-freeing"/>.
 </t>
 
-<section title="Peer Is Full">
-
 <t>
-In this case, the agent will receive connectivity checks from its
-peer. When an agent has received a connectivity check that includes
-the USE-CANDIDATE attribute for each component of a media stream, the
-state of ICE processing for that media stream moves from Running to
-Completed. When the state of ICE processing for all media streams is
-Completed, the state of ICE processing overall is Completed.
+If the peer is a full agent, the lite agent selects a candidate pair
+when the full agent nominates it.  When the lite agent has selected a
+candidate pair for all components of all media streams, the ICE session is
+Completed.
 </t>
 
 <t>
-The lite implementation will never itself determine that ICE
-processing has failed for a media stream; rather, the full peer will
-make that determination and then remove or restart the failed media
-stream as part of subsequent candidate exchange process.
+If the peer is a lite agent, the agent pairs local candidates with
+remote candidates that are for the same media stream and have the same
+component, transport protocol, and IP address family.  For each
+component of each media stream, if there is only one candidate pair,
+that pair is added to the valid list.  If there is more than one pair,
+it is RECOMMENDED that an agent follow the procedures of RFC 6724
```

Comment:
It seems that this 6724 might need more thought on how it is used. THis does seem like a non editorial change from old text.